src/libwalletqt/Wallet.h

```diff
@@ -175,6 +181,7 @@ class Wallet : public QObject
     mutable QTime   m_daemonBlockChainTargetHeightTime;
     mutable quint64 m_daemonBlockChainTargetHeight;
     int     m_daemonBlockChainTargetHeightTtl;
+    QObject * m_walletManager;
```

Comment:
not clear why do you need this?  WalletManager is a singleton currently, it's not supposed to be deleted. So PendingTransaction will never be deleted from "parent" object.